src/spell_check.c

```diff
@@ -182,16 +182,6 @@ void SpellCheck_SendReplyOnTerm(RedisModuleCtx *ctx, char *term, size_t len, RS_
 
   RS_Suggestion **suggestions = spellCheck_GetSuggestions(s);
 
-  for (int i = 0; i < array_len(suggestions); ++i) {
-    if (suggestions[i]->score == -1) {
-      suggestions[i]->score = 0;
-    } else {
-      if (totalDocNumber > 0) {
```

Comment:
Can totalDocNumber be 0? if so you do not check it now...

Reply:
No since it is number of documents in the index and we do have a result (otherwise, we wouldn't get here). 
We can assert it or add it into the code but I believe it is not possible.

Reply:
If it's not possible then let's assert it

Reply:
Added an 'of'